### ISSUE TITLE
The `inconsistent-args` works incorrectly

### ISSUE THREAD

**Issue by maxonfjvipon** [objectionary/lints#517]
Consider this XMIR:
```xml
<o base=".read" line="20" name="@" pos="10">
   <o base="Q.org.eolang.io.tee-input" line="21" pos="12">
      <o base="Q.org.eolang.io.bytes-as-input" line="22" pos="14">
         <o base="Q.org.eolang.bytes" line="22" pos="29">
            <o line="22" pos="29">01-02-03-04-05</o>
         </o>
      </o>
      <o base="Q.org.eolang.io.malloc-as-output" line="23" pos="14">
         <o base="$.m" line="23" pos="31"/>
      </o>
   </o>
   <o base="Q.org.eolang.number" line="24" pos="12">
      <o base="Q.org.eolang.bytes" line="24" pos="12">
         <o line="24" pos="12">40-14-00-00-00-00-00-00</o>
      </o>
   </o>
</o>
```

I see this: `[WARNING] org.eolang.io.tee-input-tests:20 Object '.read' has arguments inconsistency (inconsistent-args)`

It seems there's something wrong with the lint, it works incorrectly

**Comment by h1alexbel** [comment]
@maxonfjvipon failed to catch the bug, please check tests in #529

**Comment by h1alexbel** [comment]
@yegor256 let's make new release. New release should close several issues

**Comment by yegor256** [comment]
@rultor release, tag is `0.0.48`

**Comment by h1alexbel** [comment]
@maxonfjvipon please try new version: [0.0.48](https://github.com/objectionary/lints/releases/tag/0.0.48). It may solve the issue

**Comment by VasilevNStas** [comment]
@yegor256 @volodya-lombrozo @maxonfjvipon, this warning is **no longer reproducible on `master`**: the `inconsistent-args` lint belonged to the whole-program-analysis layer that was removed in commit `235cda2d` (PR #845, Apr 2026). The `tee-input` / `.read` example from this issue is a WPA case; no WPA lints exist anymore.

I'd suggest closing this as resolved-by-refactoring.

**Comment by volodya-lombrozo** [comment]
`inconsistent-args` was moved to wpa repository: https://github.com/objectionary/wpa
Moreover, it seems to be fixed. @maxonfjvipon please, feel free to open a new issue in  the wpa repo if it's still a bug
